src/auth_manifests.yaml

```
@@ -0,0 +1,71 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app: training-operator
+  name: {{ namespace }}-training-operator
```

The `training-operator` part could also change depending on the application name with which it gets deployed. I would recommend templating the application name also. Something like:
```suggestion
    app: {{ app-name }}
  name: {{ namespace }}-{{ app-name }}
```

---

That's true, good catch! I'll change it (:

---

Updated in latest version. Resolving this.